### ISSUE TITLE
"Loading repositories" hanging indefinitely

### ISSUE THREAD

**Issue by Kaos-Industries** [wbond/package_control#1103]
Is this place active? It appears to be kind of dead judging by the latest few issues.

The problem I'm having is that on Sublime Text 3, Windows 7, clicking _Package Control - Install Packages_ doesn't bring up the usual list of packages to install, but gets stuck on _Loading repositories_. Apart from startup and reloading plugins, nothing is happening in the console, even with `"debug": true` and once having selected _Install Packages_. After having loaded the plugins, the last two lines in the console show 

```
CSSLint is a .sublime-package; Verifying required files.
Package Control: Skipping automatic upgrade, last run at 2016-04-01 00:49:52, next run at 2016-04-01 01:49:52 or after
```

Tried disabling CSSLint, no change.

**Comment by wbond** [comment]
If you aren't seeing anything in the console with debug true, then you probably placed the setting in the wrong file.

Usually hangs like this are due to packages cloned via git where git is waiting on user input while checking for changes. However, a debug log is the correct way to figure out what is going on.

**Comment by Kaos-Industries** [comment]
I'm placing it in _Preferences.sublime-settings - User_?

Would that involve me using git? Because I don't.

**Comment by wbond** [comment]
Settings go under Preferences > Package Settings > Package Control > Settings – User.

**Comment by wbond** [comment]
Feel free to open a new issue with a debug log if you are still having trouble.
